frontend/workflows/projectSelector/src/hello-world.tsx

```diff
@@ -51,10 +52,15 @@ export interface State {
 
   projectData: { [projectName: string]: Project };
   loading: boolean;
+  error: ClutchError | undefined;
 }
 
-// TODO: subout with full manifest structure (from proto def)
 interface Project {
+  name: string;
+  tier: string;
+  owners: string[];
+  languages: string[];
+  data: any;
   upstreams: string[];
   downstreams: string[];
 }
```

Comment:
Why cant we use the proto here?

Reply:
@mikecutalo latest commit updates to use the Project type and also use the request/response types as well https://github.com/lyft/clutch/pull/1538/commits/cdeb56757aebe8e0304981531dd5446fa9961aad